### PR TITLE
Feat: Dockerfile

### LINKED ISSUE #17: Switch to some Async logging framework.
There are multiple ones out there, and we need performance.

### LINKED ISSUE #18: Dockerize Ducky
Provide a public Dockerfile for redundancy and constant avability. However in order to do this, we'll need either a 
- Config
- Exposed jvm value
- or a "hidden" file lilke .env

Hardcoded values won't work here if we want to scale.

### PR DESCRIPTION
**Pull request by KoxSosen**
This PR implements Docker support for Ducky, or in other worlds, it "Dockerizes" Ducky. 

Checklist

- [x] Make it actually work.
- [x] Install Java 11
- [x] Add email
- [x] Add JVM flags to the script.
- [x] Maybe add enviroment variables.

General concept:
- You build using your Ducky-<ver>.zip, and the image will take care of the rest.

Fixes #18 
Marked as Draf; This hasn't been tested yet, and it isn't production ready.

### CODE REVIEW

**Comment by KoxSosen** [comment]
Propably won't publish to docker hub, as you have to include your own build in the same dir as the Dockerfile.

**Comment by KoxSosen** [comment]
```
docker build \
-t ducky:1.6.4 \
--build-arg version=1.6.4 \
--build-arg javaversion=adoptopenjdk-11-hotspot \
.
```

**Comment by KoxSosen** [comment]
Should be ready to merge.

**Review by TTRCharlie — COMMENTED** [review]
`Step 9/14 : COPY Ducky-$BUILDVER.tar /opt/
COPY failed: file not found in build context or excluded by .dockerignore: stat Ducky-.tar: file does not exist`

`debconf: delaying package configuration, since apt-utils is not installed`

**Review by TheForgTN — APPROVED** [review]

**Comment by KoxSosen** [comment]
> `Step 9/14 : COPY Ducky-$BUILDVER.tar /opt/ COPY failed: file not found in build context or excluded by .dockerignore: stat Ducky-.tar: file does not exist`
> 
> `debconf: delaying package configuration, since apt-utils is not installed`

The Docker image works the following way: It copies the build for the bot first to the container, and then it runs it. The reason for this, is because we don't have a config file, and you have to specify a token and other parameters **before** compiling.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Using Debian 10.
```
Comment (CDFN) [inline Dockerfile]: It doesn't seem to be Debian 10 anymore :D

Comment (KoxSosen) [reply Dockerfile]: Thank you for the catch!

Comment (KoxSosen) [reply Dockerfile]: Thank you for the catch!

**Comment by KoxSosen** [comment]
Successfully ran it for 24 hours in production without any issues. Merging.

**Pull request merged**